ajax/libs/vue-amap/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vue-amap",
+  "version": "0.1.6",
+  "description": "A GaoDe Map Component Library for Vue.js.",
+  "filename": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:ElemeFE/vue-amap.git"
+  },
+  "author": "ElemeFE",
+  "homepage": "http://elemefe.github.io/vue-amap",
+  "keywords": [
+    "eleme",
+    "vue",
+    "components",
+    "amap",
+    "map"
+  ],
+  "license": "MIT",
+  "npmName": "vue-amap",
+  "npmFileMap": [
+    {
+      "basePath": "src",
```

Comment:
We need the pre-built files under `dist` folder. Please use `"dist"` instead. Thanks.